### ISSUE TITLE
Sprites are not rendering correctly in Chrome on Linux.

### ISSUE THREAD

**Issue by arthurvasconcelos** [simiancraft/create-phaser-app#10]
Linux: Ubuntu 18.04
Chrome: Version 67.0.3396.79 (Official Build) (64-bit)

![screenshot from 2018-06-06 19-13-44](https://user-images.githubusercontent.com/1286768/41067978-f69ac1c8-69bd-11e8-810d-929d7dc18399.png)

Firefox is rendering normally.

Can this be a Phaser bug? Or is something with Chrome?

**Comment by the-simian** [comment]
here is OSX: 
![screen shot 2018-06-06 at 11 24 26 pm](https://user-images.githubusercontent.com/954596/41078223-0e0e96fe-69e1-11e8-8758-693ce271ba62.png)

**Comment by the-simian** [comment]
and here is windows: 
![image](https://user-images.githubusercontent.com/954596/41078232-1e034cf8-69e1-11e8-80ed-96267d0d840f.png)

**Comment by the-simian** [comment]
You're testing with this link, right? https://simiancraft.github.io/create-phaser-app/

**Comment by arthurvasconcelos** [comment]
yeah, I tested with your gh-pages link, I re-tested now and for some reason now it's working perfectly.

![screenshot from 2018-06-07 11-50-54](https://user-images.githubusercontent.com/1286768/41107663-2cf20840-6a49-11e8-8705-60b298751a0a.png)

So, maybe it was a bug with chrome that was fixed or something with ubuntu also fixed now.

Thanks anyway for the feedback! I will follow using your boilerplate! :smile: Great work!
